examples/simple_autopilot_private/outputs.tf

```diff
@@ -37,6 +37,7 @@ output "master_kubernetes_version" {
 output "ca_certificate" {
   description = "The cluster ca certificate (base64 encoded)"
   value       = module.gke.ca_certificate
+  sensitive   = true
```

Comment:
I didn't see ca cert marked sensitive in the provider. 
https://github.com/hashicorp/terraform-provider-google/blob/9331cfe4c1636249bed8a042c7580a353b47a7ed/google/resource_container_cluster.go#L749-L753
Did you receive an error while running this?

Reply:
Yep, got this when I was trying to test it.

```
Error: Output refers to sensitive values
│ 
│   on outputs.tf line 37:
│   37: output "ca_certificate" {
│ 
│ To reduce the risk of accidentally exporting sensitive data that was intended to be only internal, Terraform requires that any root module output containing sensitive data be explicitly
│ marked as sensitive, to confirm your intent.
│ 
│ If you do intend to export this data, annotate the output value as sensitive by adding the following argument:
│     sensitive = true
```

Reply:
@rovindsouza thanks for clarifying. In that case, let's update this for all the examples.

Reply:
is this actually sensitive? this is the certificate, and not the private key used to generate the certificate, right?